src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -38,8 +38,16 @@ const SubBudgetCard = ({
   const renderCardHeader = (budgetType, budgetId) => {
     const subtitle = (
       <div className="d-flex flex-wrap align-items-center">
```

Comment:
I believe this `subtitle` could be simplified by using `Stack`, e.g.:

```jsx
<Stack direction="horizontal" gap={2.5}>
  <Badge variant="primary">Active</Badge>
  <span data-testid="offer-date">
    Expires Dec 30, 2023
  </span>
</Stack>
```

Reply:
it makes sense to me. fixed